### DISCUSSION TITLE
Problem with matching table in Digitals

### DISCUSSION THREAD

**Discussion by stephenhart8** [geovistory/.github#3] [Bugs]
While trying to create a match column from a Digital Table to entities, the matching window opens, but displays nothing, and no buttons seem to function anymore (not even the "cancel" one, I have to reload the page to exit the window).
 
<img width="1292" alt="Capture d’écran 2023-12-04 à 15 36 01" src="https://github.com/geovistory/.github/assets/49342191/7e23ac4c-0037-4840-a47d-1048642873df">

Similarly, when I try to do the matching from an already existing matching column, the window that opens is simply blank

<img width="1292" alt="Capture d’écran 2023-12-04 à 15 37 56" src="https://github.com/geovistory/.github/assets/49342191/8f657a1c-265a-4db7-848f-145c837c1655">

I imagine that the bug is related?

**Comment by valamercery** [comment]
I wasn't able to reproduce the bug. The list of mappable entities appears correctly.
![Capture du 2023-12-06 14-23-20](https://github.com/geovistory/.github/assets/51914812/70e909a9-cb58-4970-8493-3bbb0ac408ef)

**Comment by gaetanmuck** [comment]
I looked at it, found the problem, it is the same problem as in https://github.com/geovistory/toolbox/issues/326

Problem description: a cell had an annotation which had no entity, so entity label could not be found by the GUI. (A user removed the statement to the annotated entity instead of removing the annotation)

How did I solve: directly in the database, removed the statement from the cell to the annotation out of the project

Can you check if everything is solved? If yes, you can close the discussion, the problem being handled in the mentioned issue.

Question @stephenhart8,  was this conversation not supposed to be in a bug issue rather than in a discussion? (Just for my understanding)

**Comment by valamercery** [comment]
Similar problem: if the cell value is a number, the screen is greyed out but no association form is displayed.

**Comment by valamercery** [comment]
> Question @stephenhart8, was this conversation not supposed to be in a bug issue rather than in a discussion? (Just for my understanding)

This is the new procedure: first a discussion is created and then, if necessary, it is transformed into an issue, see https://github.com/geovistory#submit-feedback
